src/lib.rs

```diff
@@ -198,6 +199,18 @@ impl FastText {
             cft_fasttext_quantize(self.inner, args.inner);
         }
     }
+
+    pub fn get_word_vector(&self, word: &str) -> Vec<f32> {
```

Comment:
This can be simplified to

```rust
    pub fn get_word_vector(&self, word: &str) -> Vec<f32> {
        let c_text = CString::new(word).unwrap();
        let dim = self.get_dimension() as usize;
        let mut v = Vec::with_capacity(dim);
        unsafe {
            cft_get_word_vector(self.inner, c_text.as_ptr(), v.as_mut_ptr());
            v.set_len(dim);
        }
        v
    }
```

Reply:
Do you mind make this change?

Reply:
Ah there is a `set_len`, didn't notice that.
I can never remember std libs after all.

Reply:
The [`nomicon`](https://doc.rust-lang.org/nightly/nomicon/ffi.html) has some good resource on FFI.